src/github.com/getlantern/autoupdate-server/server/signature.go

```diff
@@ -0,0 +1,85 @@
+package server
+
+import (
+	"crypto"
+	"crypto/rand"
+	"crypto/rsa"
+	"crypto/x509"
+	"encoding/hex"
+	"encoding/pem"
```

Comment:
Thanks! it seems like I could borrow the `LoadPKFromFile` from keyman but I'll have no use for most of the other functions. I currently use just two crypto-related functions (reading key and signing a checksum) so maybe it's not that necessary at this time.